README.md

```diff
@@ -1,32 +1,249 @@
+method is called on the corresponding thread object.
+
+```java
+public static void main(String[] args) {
+    System.out.println("Start " + " (" + Thread.currentThread().getName() + ")" );
+    int treats = 4;
+    Dog snoopy = new Dog("Snoopy", false);       
+    Dog fifi = new Dog("Fifi", true);            
+        
+    Thread thread0 = new Thread( () -> snoopy.eatTreats(treats) );
+    Thread thread1 = new Thread( () -> fifi.eatTreats(treats) );
+    Thread thread2 = new Thread( () -> snoopy.takeBath() );
+
+    thread0.start();
+    thread1.start();
+    thread2.start();
+    
+    System.out.println("Finish " + " (" + Thread.currentThread().getName() + ")" );
+}  
+```
+
+By default, Java will name the threads main, Thread-0, Thread-1, and Thread-2.
+Because there are 4 threads, the order of the print statements may differ each time the program executes.
+
+For example, we might see this output:
+
+```text
+Start  (main)
+Finish  (main)
+After bath Snoopy wagging is false (Thread-2)
+Fifi eats treat #1 (Thread-1)
+Fifi eats treat #2 (Thread-1)
+Snoopy eats treat #1 (Thread-0)
+Fifi eats treat #3 (Thread-1)
+Fifi eats treat #4 (Thread-1)
+Snoopy eats treat #2 (Thread-0)
+Snoopy eats treat #3 (Thread-0)
+Snoopy eats treat #4 (Thread-0)
+After treats Fifi wagging is true (Thread-1)
+After treats Snoopy wagging is true (Thread-0)
+```
+
+The sample output corresponds to the shared timeline shown below.
+We can see the print statements in the `main` thread both happen before
+`Thread-2` executes the print statement from the call `snoopy.takeBath()`.
+`Thread-0` and `Thread-1` then alternate executing the code
+from the method calls `snoopy.eatTreats(treats)` and `fifi.eatTreats(treats)`.
+
+```text
+main       - -  
+Thread-0                -      - - -    -  
+Thread-1           - -     - -        -  
+Thread-2        -  
+```
+
+How does this happen?  Recall that each thread has its own stack.
+The image below depicts the heap and stacks for the sample program.
+Note the heap would also contain objects for each thread
+and lambda implementation, but only the dog objects are shown.
+
+
+![Multithreaded stacks](https://curriculum-content.s3.amazonaws.com/java-mod-4-threads-intro/introduction-to-threads/multithreaded.png)
+
+The Java runtime system allows the threads to take turns executing
+the methods on their stack. The order and duration of each turn
+may differ every time the program executes.
 
 ## Conclusion
 
-A short one or two paragraph summary of the contents of the lessons, recapping
-the learning goals.
+This lesson introduced the concept of concurrency by demonstrating a program with
+multiple threads. 
+
+Multithreading is the reason the browser can react to mouse clicks while loading the page,
+and why a game can handle user input from the keyboard, mouse, and touch screen while
+showing animation on the monitor screen.
+
+Once we take a deeper dive into threads, you’ll be able to handle multiple threads in your program!
```

Comment:
Maybe use the pronoun "we" here instead since it seems like the rest of the lesson takes that pronoun on?

Reply:
I deleted the sentence.